src/datachain/model/ultralytics/pose.py

```diff
@@ -102,8 +102,10 @@ def from_results(results: list["Results"]) -> "YoloPoses":
                 cls.append(s["class"])
                 names.append(name)
                 confidence.append(s["confidence"])
-                box.append(BBox.from_dict(s.get("box", {}), title=name))
-                pose.append(Pose3D.from_dict(s.get("keypoints", {})))
+                if s.get("box"):
```

Comment:
How about `if "box" in s:` instead?

Reply:
apparently ultralytics returns `None` for some (or all of these objects). Easier to ignore them in such cases